docs/section1/Rule1-2.md

```diff
@@ -0,0 +1,43 @@
+# Section 1 - Rule 1-2
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** True  
+**Rule ID:** 1-2  
+**Rule Description:** The performance of the proposed design is calculated in accordance with Standard 90.1-2019 Appendix G, where Performance Cost Index = Proposed building performance (PBP) /Baseline building performance (BBP), where both the PBP and the BBP include all end-use load components associated with the building when calculating the Performance Cost Index (PCI).  
```

Comment:
I am a little confused by this rule since the calculation of PCI does not change whether or not renewables are greater than 5%. I think we should discuss this.